src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -71,6 +71,28 @@ public void close() throws SQLException {
         return dataSets;
     }
 
+    boolean insert(String tableName, DataSet dataset) {
+        String dataSetFormatted = dataSetFormatted(dataset);
+        String sqlQuery = "INSERT INTO " + tableName + " VALUES(" + dataSetFormatted + ")";
```

Comment:
Faster than 'format' but not so readable, isn't it?

Reply:
May be( Will be fixed

Reply:
In that case, when parameters goes sequentially I don't agree that 'format' will be much readable...